### ISSUE TITLE
Will this work with Meteor 1.2

### ISSUE THREAD

**Issue by suhail-ansari-apconic** [MeteorPackaging/Font-Awesome#5]
Since the less package has been changed will this work with Meteor 1.2

**Comment by rosspeoples** [comment]
I just tried using it with Meteor 1.2 and the answer is no.

Here's the error I get when I install it:

```
=> Errors prevented startup:

   While reading package from
   `/Users/me/Code/meteor/packages/fontawesome`:
   package.js:13:7: The `isAsset` option to `addFiles` is deprecated. Use PackageAPI#addAssets
   instead.

=> Your application has errors. Waiting for file change.
```

**Comment by splendido** [comment]
just published version `4.4.0_1`
thanks!
